packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -244,20 +244,20 @@ module.exports = {
+            { text: 'Twitter', link: 'https://twitter.com/OktaDev', target: '_self', icon: '<svg width="18" height="18" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M9 0a9.001 9.001 0 000 18A9.001 9.001 0 009 0zm4.11 7.017c.003.089.005.178.005.267 0 2.73-2.078 5.878-5.877 5.878a5.847 5.847 0 01-3.167-.928 4.144 4.144 0 003.058-.856A2.068 2.068 0 015.2 9.943a2.056 2.056 0 00.934-.035 2.066 2.066 0 01-1.657-2.051c.278.154.597.247.935.258a2.064 2.064 0 01-.64-2.758A5.865 5.865 0 009.03 7.515a2.066 2.066 0 013.52-1.884c.47-.092.913-.264 1.312-.5a2.074 2.074 0 01-.909 1.142 4.12 4.12 0 001.187-.326 4.2 4.2 0 01-1.03 1.07z"/></svg>' },
+            { text: 'YouTube', link: 'https://www.youtube.com/c/oktadev', target: '_self', icon: '<svg width="18" height="18" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M7.879 10.372l2.928-1.686L7.878 7v3.372z"/><path d="M9 0a9.001 9.001 0 000 18A9.001 9.001 0 009 0zm5.624 9.009s0 1.825-.232 2.705a1.41 1.41 0 01-.991.992c-.88.231-4.401.231-4.401.231s-3.511 0-4.4-.24a1.41 1.41 0 01-.992-.992C3.376 10.835 3.376 9 3.376 9s0-1.825.232-2.705c.13-.482.519-.871.991-1.001C5.48 5.062 9 5.062 9 5.062s3.52 0 4.4.241c.482.13.862.51.992.992.241.88.232 2.714.232 2.714z"/></svg>' },
+            { text: 'Forum', link: 'https://devforum.okta.com/', target: '_self', icon: '<svg width="19" height="19" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M9.434.158a8.927 8.927 0 00-6.51 3.102C.098 6.513.012 11.48 2.725 14.81a9.033 9.033 0 009.974 2.81c1.454-.51 3.01-.174 4.061.878l.297.297.296-.296a2.854 2.854 0 00.002-4.032l-.232-.232c2.437-3.538 2.016-8.4-1.04-11.457A9.014 9.014 0 009.434.158zm3.189 6.08a4.107 4.107 0 01-.008 5.801 4.106 4.106 0 01-5.801.008 4.106 4.106 0 01.007-5.802 4.107 4.107 0 015.802-.007z"/></svg>' },
+            { text: 'Blog RSS', link: 'https://developer.okta.com/feed.xml', target: '_self', icon: '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><path d="M50 5C25.1 5 5 25.1 5 50s20.1 45 45 45 45-20.1 45-45S74.9 5 50 5zM34.7 73.5c-3.9 0-7-3.2-7-7 0-3.9 3.2-7.1 7-7.1 3.9 0 7.1 3.2 7.1 7.1 0 3.8-3.2 7-7.1 7zm22.4-.4c-2.2 0-4.2-1.6-4.5-3.8-.8-5.2-3.3-9.9-7.1-13.7-3.8-3.8-8.5-6.2-13.6-7.1-2.2-.4-3.8-2.3-3.8-4.5 0-2.8 2.6-5 5.4-4.5 14.4 2.4 25.8 13.8 28.3 28.3.4 2.7-1.8 5.3-4.7 5.3zM70.8 69c-2.2-20.3-18.2-36.5-38.4-38.6-2.3-.2-4.1-2.2-4.1-4.6 0-2.7 2.3-4.9 5-4.6C57.8 23.7 77.5 43.3 80 68c.3 2.7-1.9 5-4.6 5-2.4.1-4.4-1.7-4.6-4z" fill="#dd2864"/></svg>' },
```

Comment:
Do we need to target _self for all of these? It seems reasonable to also open new tabs for these links, especially non-okta domains.

Reply:
These were being opened in the current tab before, so I kept these as is + the "external" icon automatically gets added through css when the target attribute is set to `_blank`.